jekyll/_cci2_ja/pipeline-variables.md

```diff
@@ -1,44 +1,30 @@
 ---
 layout: classic-docs
-title: "パイプライン変数"
-short-title: "パイプライン変数"
-description: "パイプラインの変数、パラメーター、値についての詳細情報"
+title: "Pipeline values and parameters"
+description: "Detailed information about pipeline parameters and values"
 categories:
   - はじめよう
 order: 1
 version:
   - クラウド
+  - Server v3.x
 ---
 
-パイプライン変数を使用すると、再利用可能なパイプラインを構成できます。 パイプライン変数を使用するには、[パイプライン]({{ site.baseurl }}/2.0/build-processing)を有効化し、設定ファイルで[バージョン]({{ site.baseurl }}/2.0/configuration-reference/#version) `2.1` 以上を指定する必要があります。
-
-パイプライン変数には、次の 2 つの種類があります。
+Pipeline values and parameters can be used to create reusable pipeline configurations.
```

Comment:
[confirmation required]

this localization looks to be missing 🤔

Reply:
[update]

discussed with Mayuko-san and we will tackle some of these 🇺🇸 strings in later PRs (else this PR will be too big).